subsys/bluetooth/mesh/keys.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
We do not have `internal` suffix for internal headers. They are just located in the same place as the source files. Then just including then as local headers highlights it is internal or doesn't. Even more, stack sources and `internal` headers do not include `external` headers explicitly only via `mesh.h`. There is no direct path to these headers.

No one `private` header has guard definitions. It helps to avoid to including headers within the headers to have a deal with strange interfile dependencies. Instead of this all headers are included only in source files in a sequence they depend on each other. It allows to not bother about guards. Of course, there are a couple of headers included inside headers and these are actual bugs that sneaked through review. :(